client/src/VuDLPrep.jsx

```diff
@@ -22,18 +29,14 @@ class VuDLPrep extends React.Component{
             + encodeURIComponent(job) + extra;
     }
 
-    selectJob = (category, job) => {
-        this.selector.hide();
-        this.paginator.loadJob(category, job);
-    }
-
     ajax = (params) => {
         params.beforeSend = function (xhr) {
             xhr.setRequestHeader('Authorization', 'Token ' + this.props.token);
         }.bind(this);
         $.ajax(params);
     }
 
+    // TODO: Why does this one need url when getJobUrl and getImageUrl don't?
```

Comment:
This needs a url parameter because it's a shared utility function for retrieving JSON, and it is called in different places to pull different JSON documents from different API endpoints.

Reply:
I stuffed general helper routines into the VuDLPrep component because of the way React and Rails interfaced. Since we're doing this in pure JS now, it may make sense to create separate helper classes and import them where they are needed, instead of wiring components together unnecessarily.